gcp/vm/vars.tf

```diff
@@ -0,0 +1,67 @@
+variable "env" {
+    description = "The environment it will serve (prod, staging, etc)"
+    type = "string"
+}
+
+variable "project_name" {
+    description = "GCP project name"
+    type = "string"
+}
+
+variable "region" {
+    description = "GCP region"
+    default = "us-central1"
+}
+
+variable "instance_count" {
+    description = "Number of instances to create"
+    default = 1
+}
+
+variable "instance_name" {
+    type = "string"
+}
+
+variable "instance_type" {
+    type = "string"
+    default = "n1-standard-1"
+}
+
+variable "instance_image" {
+    type = "string"
+    default = "centos-7-v20190423"
+}
+
+variable "instance_project" {
+    type = "string"
+    default = "centos-cloud"
+}
+
+variable "network" {

```

Comment:
Having the subnetwork, you can search for its parent network with:
```hcl
data "google_compute_subnetwork" "my-subnetwork" {
  name   = "${element(var.subnetworks, 0)}"
  region = "${var.region}"
}
```
Although it seems to be an over-engineering approach, it avoids human error and typos coming from the module "parameters". Also, removes one required (without default value) variable, which keeps a smaller contract.

Reply:
Got his point. There is no need to provide a network ID that can be calculated from any given subnet (your parent network) in the same region. However, by providing a name argument to the google_compute_subnetwork resource, this module will be limited in provisioning every VM on a single subnet rather than randomize on available subnetworks. Therefore, I will apply this data source only to infer the parent network name (and dropping network variable), keeping subnetwork randomize on instance resource. Fair enough?

Reply:
Actually the point was to just get the parent network "${data.google_compute_subnetwork.my-subnetwork.network}" and use it for the `network_interface` instead of having the variable passed through "parameter" (:

I've seem that you already did this, and it's great :+1: